commands/bridge_token_rm.go

```diff
@@ -0,0 +1,29 @@
+package commands
+
+import (
+	"github.com/spf13/cobra"
+
+	"github.com/MichaelMure/git-bug/bridge/core"
+)
+
+func runBridgeTokenRm(cmd *cobra.Command, args []string) error {
+	err := core.RemoveToken(repo, args[0])
```

Comment:
Here again, a single function in the core would simplify this. As it is, you also have a bug: if attempting to remove an non-existing local token, `RemoveToken()` should return an error.